hardware/gpsPi/gps_reader.py

```diff
@@ -49,7 +48,7 @@ def get_geolocation(self):
                 "latitude": latitude_decimal,
                 "longitude": longitude_decimal,
             }
-            data["date"] = datetime.now(timezone.utc).astimezone().isoformat()
+            data["date"] = date_str_with_current_timezone()
```

Comment:
I think @MichaelLally made this change in his previous PR, are we sure that it should be changed back?

Reply:
Yes, I asked him. He said we can.

Reply:
oh great!